seedDB change and update

abc-1211 opened the pull request.

abc-1211 committed:
seedDB change and update

JoeDodgson commented on scripts/seedsDB.js:
```diff
@@ -7,40 +7,40 @@ mongoose.connect(process.env.MONGODB_URI || "mongodb://localhost/engineerdb");
-    client: "client1",
-    teamMembers: ["member1", "member3", "member5"],
+    client: "British Government",
+    teamMembers: ["Joe", "ian", "dan"],
```
teamMembers needs to be of the format:
```
teamMembers: [{
     name: "Joe Dodgson",
     _id: "...............",
}]
```
as set by you on the new project page
![image](https://user-images.githubusercontent.com/50573262/92726055-f02d4800-f364-11ea-861c-1b7dad4954ad.png)

Name needs to include first name and last name.
Also, all names should be capitalised.
And these should be names and _id's which are present in the database

JoeDodgson commented on scripts/seedsDB.js:
```diff
@@ -7,40 +7,40 @@ mongoose.connect(process.env.MONGODB_URI || "mongodb://localhost/engineerdb");
     location: {
-      lat: 52.506752,
+      lat: 48.6345,
       lng: -1.927523,
```
This location is somewhere in France. Please update with the location of London bridge

JoeDodgson commented on scripts/seedsDB.js:
```diff
@@ -7,40 +7,40 @@ mongoose.connect(process.env.MONGODB_URI || "mongodb://localhost/engineerdb");
-      lat: 52.509752,
-      lng: -1.923867,
+      lat: 47.4266,
+      lng: -1.463867,
```
Please update with the location of Big Ben

JoeDodgson commented on scripts/seedsDB.js:
```diff
@@ -7,40 +7,40 @@ mongoose.connect(process.env.MONGODB_URI || "mongodb://localhost/engineerdb");
-      lat: 52.505333,
-      lng: -1.923352,
+      lat: 50.353626,
+      lng: 1.6235,
```
Please update with the location of London Eye

JoeDodgson commented on scripts/seedsDB.js:
```diff
@@ -58,49 +58,49 @@ db.Project.remove({})
         riskId: "riskId1",
-        description:"discription1",
+        description: "Annual maintenance",
         designDiscipline: "designDiscipline1",
```
Update design discipline with something relevant. E.g. 'Structures'

JoeDodgson commented on scripts/seedsDB.js:
```diff
@@ -58,49 +58,49 @@ db.Project.remove({})
+        title: "London Eye",
         riskId: "riskId1",
-        description:"discription1",
+        description: "Annual maintenance",
```
Needs to be a risk, rather than a project description. E.g. "Risk of structural failure of spokes on London eye during maintenance"

JoeDodgson commented on scripts/seedsDB.js:
```diff
@@ -58,49 +58,49 @@ db.Project.remove({})
-        projectId:"5f4eec89cc853f32ec03243e"
+        projectId:"5f59f053a94acd5aa0e4ebed"
     },
   
```
Can you add some more risks which have a range of likelihood and severity values, so that we can demonstrate the colour coding of risks on the map

JoeDodgson commented on scripts/seedsDB.js:
```diff
@@ -7,40 +7,40 @@ mongoose.connect(process.env.MONGODB_URI || "mongodb://localhost/engineerdb");
-    client: "client2",
-    teamMembers: ["member1", "member2", "member4"],
+    client: "British Government",
+    teamMembers: ["Ben", "Mary", "Sarah"],
```
teamMembers needs to be an array of objects

JoeDodgson commented on scripts/seedsDB.js:
```diff
@@ -7,40 +7,40 @@ mongoose.connect(process.env.MONGODB_URI || "mongodb://localhost/engineerdb");
-    client: "client1",
-    teamMembers: ["member1", "member3", "member5"],
+    client: "British Government",
+    teamMembers: ["Kenny", "Queenie", "David"],
```
teamMembers needs to be an array of objects

JoeDodgson commented on scripts/seedsDB.js:
```diff
@@ -58,49 +58,49 @@ db.Project.remove({})
+        title: "London Bridge",
         riskId: "riskId2",
-        description:"discription2",
+        description: "Road repair",
```
This should be a risk description, not an activity. E.g. risk of worksite staff being struck by traffic during maintenance

abc-1211 committed:
SeedsDB updated

abc-1211 committed:
SeedsDB update

JoeDodgson committed:
Amendments to seed data to address my own comments

-Corrected console logs and comments
-Added more descriptive data with accurate locations

JoeDodgson reviewed (approved):
I have made amendments myself to address outstanding comments
